src/__tests__/withAuthUser.test.js

```diff
@@ -897,6 +900,112 @@ describe('withAuthUser: rendering/redirecting', () => {
     )
     expect(container.firstChild).toBeNull()
   })
+
+  it('calls the "authPageURL" function with an undefined context and unauthed AuthUser if redirecting to the login outside the base path on client', () => {
+    expect.assertions(3)
+    const withAuthUser = require('src/withAuthUser').default
+    const MockSerializedAuthUser = undefined // no server-side user
+    useFirebaseUser.mockReturnValue({
+      ...getUseFirebaseUserResponse(),
+      user: undefined, // no client-side user
+      initialized: true, // already initialized
+      authRequestCompleted: true,
+    })
+    const mockConfig = getMockConfig()
+    let propsSpy
+    setConfig({
+      ...mockConfig,
+      authPageURL: (props) => {
+        propsSpy = props
+        return {
+          destination: `/some-auth-page`,
+          permanent: false,
+          basePath: false,
+        }
+      }, // custom auth page
+    })
+    const MockCompWithUser = withAuthUser({
+      whenUnauthedBeforeInit: AuthAction.RENDER,
+      whenUnauthedAfterInit: AuthAction.REDIRECT_TO_LOGIN,
+      whenAuthed: AuthAction.RENDER,
+    })(MockComponent)
+    render(
+      <MockCompWithUser
+        serializedAuthUser={MockSerializedAuthUser}
+        message="How are you?"
+      />
+    )
+    expect(propsSpy.ctx).toBeUndefined()
+    expect(propsSpy.AuthUser.id).toBeNull()
+    expect(window.location.assign).toHaveBeenCalledWith('/some-auth-page')
+  })
+
+  it('calls the "appPageURL" function with an undefined context and unauthed AuthUser if redirecting to the login outside the base path on client', () => {
```

Comment:
Fix test description. This is testing the object URL, not a function.

Reply:
This one was both 😆